articles/how-to-compile-bitcoin-core-and-run-the-tests.html

```diff
@@ -93,8 +93,8 @@
                       installed):
                       <code>
                         brew install automake berkeley-db4 libtool boost
-                        miniupnpc openssl pkg-config protobuf python qt libevent
-                        qrencode ccache
+                        miniupnpc pkg-config python qt libevent qrencode
+			ccache
```

Comment:
(Sorry to be picky) please use spaces, not tabs :)

Reply:
(Generally, use the formatting that is already used in the document.)

Reply:
Sorry, my oversight!